### PR TITLE
feat(manage-user): update to new roles&perm

### PR DESCRIPTION
**Pull request by urbanisierung**
/relates https://github.com/camunda-cloud/console/issues/2581
/closes https://github.com/camunda-cloud/camunda-cloud-documentation/issues/451

### CODE REVIEW

File: docs/components/cloud-console/manage-organization/manage-users.md [hunk 1]
```diff
@@ -12,24 +12,22 @@ If several users need access to the same Zeebe cluster, all users can be assigne
...
+- **Task User**: Full access to Tasklist and read-only access to Clusters
+- **Developer**: Full access to Console, except deletion privileges. Full access to Operate, and Tasklist
+- **Visitor**: Read-only access
+- **Analyst**: Full access to Optimize and read-only access to Clusters
 
 Users are invited to a Camunda Cloud organization via their email address, which must be accepted by the user. As long as the invitation has not been accepted, the user remains in the `Pending` state.
 
-People who do not yet have a Camunda Cloud account can also be invited to an organization. To access the organization, however, the invited individual must first create a Camunda Cloud account.
\ No newline at end of file
+People who do not yet have a Camunda Cloud account can also be invited to an organization. To access the organization, however, the invited individual must first create a Camunda Cloud account.
```
Comment (ultraschuppi) [inline docs/components/cloud-console/manage-organization/manage-users.md:33]: ...by following the instructions in the invitation email?

File: docs/components/cloud-console/manage-organization/manage-users.md [hunk 2]
```diff
@@ -12,24 +12,22 @@ If several users need access to the same Zeebe cluster, all users can be assigne
 
 ## Users
 
-Under this setting, members of the current organization can be managed. A user can have one of the following roles:
+The first user in an organization is usually the owner of the organization. An owner has all rights in an organization and can manage all settings accordingly. An organization cannot have more than one owner.
```
[the remark this answers is not in the document]
Comment (urbanisierung) [reply docs/components/cloud-console/manage-organization/manage-users.md]: fixed

File: docs/components/cloud-console/manage-organization/manage-users.md (hunk 1 above)
Comment (urbanisierung) [reply docs/components/cloud-console/manage-organization/manage-users.md]: fixed

**Comment by nazlikaya** [comment]
@urbanisierung 

- [ ] Can you reorder the role explanation list as in the application: Operations Engineer, Analyst, Task User, Developer, Visitor.

- [ ] The text also needs to tell a user can be assigned multiple roles.

**Comment by urbanisierung** [comment]
Hey @christinaausley, Cloud Console introduces a new roles and permission managenment so we need to update the corresponding docs page. Can you please review the PR? Thanks!

**Review by christinaausley — DISMISSED** [review]
Really like how you've clarified this. Looks good from a technical writing perspective!

File: docs/components/cloud-console/manage-organization/manage-users.md [hunk 3]
```diff
@@ -12,24 +12,24 @@ If several users need access to the same Zeebe cluster, all users can be assigne
...
+In addition to the owner, the **Admin** role is available as a second role with comprehensive rights. The admin role has the same rights as the owner, with the difference that an admin cannot manage other admins.
+
+The following roles are additionally available, providing dedicated rights for specific elements in Camunda Cloud.
+
+- **Operations Engineer**: Full access to Console and Operate, except cluster deletion privileges
+- **Analyst**: Full access to Optimize and read-only access to clusters
+- **Task User**: Full access to Tasklist and read-only access to clusters
+- **Developer**: Full access to Console, except deletion privileges; full access to Operate and Tasklist
+- **Visitor**: Read-only access
+
```
Comment (felix-mueller) [inline docs/components/cloud-console/manage-organization/manage-users.md:30]: @urbanisierung could we explain Web Modeler here too?

Comment (urbanisierung) [reply docs/components/cloud-console/manage-organization/manage-users.md:30]: @felix-mueller is `Web Modeler` a new role? Currently the backend only supports the following _new_ roles:

```
export enum OrganizationRole {
  MEMBER = 'member', // legacy
  ADMIN = 'admin',
  OWNER = 'owner',
  SUPPORTAGENT = 'supportagent',
  OPERATIONS_ENGINEER = 'operationsengineer',
  TASK_USER = 'taskuser',
  ANALYST = 'analyst',
  DEVELOPER = 'developer',
  VISITOR = 'visitor',
}
```

Comment (felix-mueller) [reply docs/components/cloud-console/manage-organization/manage-users.md:30]: according to https://docs.google.com/document/d/1FZYFn9fK2sn6ksjh7drSNad1vyMEIwK6p4KlG353yfM/edit#heading=h.aw86sy4n19y7 most roles are able to access Modeler. I was just wondering if we want to clarify this in the docs here 
@CatalinaMoisuc any thoughts? do you cover this elsewhere in the docs?

**Review by christinaausley — DISMISSED** [review]

**Comment by akeller** [comment]
FYI this is still a draft PR. Are you looking to merge into `master` or `release-1.40.`? Currently, it's targeting `master`.

**Comment by ultraschuppi** [comment]
> FYI this is still a draft PR. Are you looking to merge into `master` or `release-1.40.`? Currently, it's targeting `master`.

yea, i usually just merge to push it to prod directly after. if it is helpful for you to get this into another branch (so we can have a coorinated rollout for 1.4) - i can do that as well

@santanaluiz i assume we are pretty good on the gutfeeling-confidence-level area? ;)

**Comment by santanaluiz** [comment]
that's right @ultraschuppi

**Comment by akeller** [comment]
@ultraschuppi please merge to `release-1.4.0` when ready

**Comment by ultraschuppi** [comment]
> @ultraschuppi please merge to `release-1.4.0` when ready

replacing this PR with https://github.com/camunda-cloud/camunda-cloud-documentation/pull/723

**Pull request closed without merging**